### DISCUSSION TITLE
LED2 is used as both Power and Link status

### DISCUSSION THREAD

**Discussion by mvadu** [arendst/Tasmota#12836] [Q&A] [answered]
I bought a plug recently and found it had ESP8266 but the device wasn't in the tasmota DB before. I added a [PR ](https://github.com/tasmota/docs/pull/780) for the module and submitted the form for the device as well. 

It has two LEDs, and per the PCB tracing I can see GPIO0 - Red LEDi and GPIO2 - Blue LEDi. 

This is the template. 
{"NAME":"Linganzh SWA5","GPIO":[320,0,321,0,0,0,0,0,0,32,0,224,0,0],"FLAG":0,"BASE":18}

But only blue LED is flashing for Wifi/mqtt errors as well as showing the relay power state. Per [docs](https://tasmota.github.io/docs/Lights/#status-leds), "If more than one LED component is defined, Led1/Led1i will act as the Wi-Fi/MQTT status LED and the next defined LED (e.g., Led2/Led2i) will act as the LED that indicates the power state of the relay(s). ".  I was expecing red led to flash for wifi issues, and blue for the replay state. What am I doing wrong?

**Comment by ascillato2** [comment, marked as answer]
> I was expecting red led to flash for wifi issues,

use for GPIO0 definition `LEDLINK_i` instead of LED1_i

> and blue for the replay state.

use for GPIO2 definition `LED1_i`

In the LED**_X_** the number **_X_** is the relay.

LEDLINK is for MQTT and WIFI indications ONLY

Recommendation:
GPIO2 - LED BLUE - Use LEDLINK_i
GPIO0 - RED LED - Use LED1_i

since most devices uses the blue for communications and the red for power indication.

**Reply by mvadu** [reply]
Thank you!! I missed the LEDLink option. This approach worked for me. 

```
{"NAME":"Linganzh SWA5","GPIO":[320,0,576,0,0,0,0,0,0,32,0,224,0,0],"FLAG":0,"BASE":18}
```

**Reply by ascillato2** [reply]
Great!!

Can you add it to the templates repository? https://templates.blakadder.com/new.html

Thanks

**Reply by mvadu** [reply]
I did submit it via google form yesterday with the older template. Once its approved I will submit a PR for this template. https://github.com/blakadder/templates seems to be used for editing than adding new devices.
